handlers/feedback.go

```diff
@@ -72,6 +76,13 @@ func (f *Feedback) AddFeedback() http.HandlerFunc {
 
 func addFeedback(w http.ResponseWriter, req *http.Request, rend interfaces.Renderer, emailSender email.Sender, from, to, lang, siteDomain string, cacheService *cacheHelper.Helper) {
 	ctx := req.Context()
+	cfg, err := config.Get()
+
+	if err != nil {
+		log.Error(ctx, "unable to retrieve service configuration", err)
+		os.Exit(1)
+	}
```

Comment:
This would be better passed in rather than retrieved at this point.

Reply:
Done